pushType.deque

```diff
@@ -0,0 +1,3 @@
+0! '0'!
+!'I have a crush on you' !printType
+0.1! 0.2! +! print!
```

Comment:
Whatever code editor you're using messed up the formatting—this file should have a trailing newline.